### ISSUE TITLE
`parallel3d_vec` projections mysteriously zero

### ISSUE THREAD

**Issue by jacobhinkle** [astra-toolbox/astra-toolbox#40]
Hello,

First, thank you for your work. A project like ASTRA has been sorely needed for a long time now.

I am having some problems with the `parallel3d_vec` geometry.  Below, I am trying to project a simple volume containing zeros outside a small box of ones.  The geometry I list below should correspond to rotating about the `y` axis as opposed to `z`.

```python
import astra
import numpy as np
import matplotlib.pyplot as plt

# set up simple example volume
sh = [25,35,45]
vol0 = np.zeros(sh)
# only one little odd-shaped block of ones
vol0[5:15,10:34,5:41] = 1

# initialize vol_geom
vol_geom = astra.create_vol_geom(sh[1], sh[2], sh[0])

# initialize proj_geom
shuv = [100, 100]
theta = np.arange(90)*np.pi/180.0
c = np.cos(theta)
s = np.sin(theta)
v = np.zeros((theta.size,12))
v[:,0] = s
v[:,1] = 0
v[:,2] = c
v[:,3:6] = 0
v[:,6] = c
v[:,7] = 0
v[:,8] = -s
v[:,9] = 0
v[:,10] = 1
v[:,11] = 0
proj_geom = astra.create_proj_geom('parallel3d_vec', shuv[0], shuv[1], v)

# get a projector id
proj_id = astra.create_projector('cuda3d', proj_geom, vol_geom)

# project the volume
pv_id, pv = astra.create_sino3d_gpu(vol0, proj_geom, vol_geom)

plt.imshow(np.squeeze(vol0[:,sh[1]//2,:]), cmap='gray', interpolation='nearest')
plt.title('Volume Slice')
plt.figure()
plt.imshow(np.squeeze(np.max(pv[:,:,:], 1)), cmap='gray', interpolation='nearest')
plt.title('MIP of Projections (all zero)')
print "Scalar range of pv is {} to {}".format(np.min(pv), np.max(pv))

# clean up
astra.data3d.delete(pv_id)
astra.projector.delete(proj_id)
```
When I run this code, the projection array `pv` is all zeros.  I also get all zeros if I backproject using this geometry with a non-zero sinogram array.

Since my detector is still centered at zero, and the `r`, `u`, and `v` vectors are all orthogonal, I am having trouble understanding how it's possible to get a zero projection.  In fact even if I'm screwing up and the ray direction is in the plane of the detector it still would surprise me to have a zero line integral, since the volume is non-zero at the origin.

Thank you for any help you can provide!
-Jacob

**Comment by adler-j** [comment]
What version of astra are you using? I had this issue and it was fixed with issue #18.

**Comment by jacobhinkle** [comment]
@adler-j You nailed it. I was running 1.7.1. Upgrading to HEAD makes the code above work as expected.
